openapi/src/main/java/io/micronaut/openapi/visitor/SchemaDefinitionUtils.java

```diff
@@ -1722,6 +1723,11 @@ public static void processSchemaProperty(VisitorContext context, TypedElement el
                 required = true;
             }
 
+            // If inclusion is ALWAYS, then all properties are required
+            if (!required && ConfigUtils.getInclude(context) == JsonInclude.Include.ALWAYS) {
```

Comment:
I don't think, that is enought... There are clear priorities in code processing (from least to most):

1. Interpretation of code without annotations
2. Annotations by Microsoft and Jackson
3. Annotations by Swagger

That is, if the value was calculated as mandatory in the first and second stages, but the Swagger annotation states that the property is not mandatory - the value from the Swagger annotation should be taken.

The same with Jackson annotations - you can override the default behavior for a specific class or property using the JsonInclude annotation at the class or property level.

I say this because more tests clearly need to be written and the feature is not as simple as it might seem at first glance